packages/next-server/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@kununu/next-server",
+  "version": "1.0.0-beta.13",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "./build.sh",
+    "prepublish": "npm run build",
+    "postpublish": "git clean -fd"
```

Comment:
This seems a bit opinionated 🤔 What if I don't want my untracked files cleaned?

Reply:
You're right. I added a clean script like we've in the kununu-ui.